packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -1,17 +1,7 @@
 .offcanvas-editor__appender .block-editor-inserter__toggle {
-	background-color: #1e1e1e;
-	color: #fff;
 	margin: $grid-unit-10 0 0 24px;
-	border-radius: 2px;
-	height: 24px;
 	min-width: 24px;
-	padding: 0;
-
-	&:hover,
-	&:focus {
-		background: var(--wp-admin-theme-color);
-		color: #fff;
-	}
+	padding: 6px 6px 6px 0;
```

Comment:
Do we need this? It doesn't seem to affect anything when I toggle it off in dev tools 🤔